torch/distributed/distributed_c10d.py

```diff
@@ -1197,6 +1198,11 @@ def init_process_group(
             _store_based_barrier(rank, store, group_name, world_size, timeout)
 
 
+# We need a synchronized integer ("color") for split communicators to
+# rendeszvous with when creating new groups.  TODO: Move this
+# somewhere and/or not be backend-specific?
```

Comment:
Synchronized between? Can the comment specify it?

Reply:
whoops, we don't even need this anymore.  removing.